mikrotik-backup/backup.sh

```diff
@@ -2,11 +2,11 @@
 # usage:
 # ./hamwan_routers.sh | ./backup.sh
 
-DIR=$(dirname "${BASH_SOURCE[0]}")/backup
+DIR=/srv/router-backup
```

Comment:
This was based on the pwd so that it could be tested non-destructively from another directory. It would be nice to keep that feature.

Maybe you could set the directory another way if you want to be able to define a default:
```suggestion
DIR=${DIR:-/srv/router-backup}
```